### ISSUE TITLE
Edit on GitHub links are broken

### ISSUE THREAD

**Issue by falexwolf** [scverse/scanpy#437]
Hi @flying-sheep,

it seems that the links broke in the course of the change to `scanpydoc`. Currently, what should resolve as
```
https://github.com/theislab/scanpy/blob/master/scanpy/tools/_paga.py#L13-L108
```
resolves as
```
https://github.com/theislab/scanpy/tree/master/tools/__init__.py#L13-L108
```
It's evidently just missing the `scanpy/` subdirectory and the fact that it shouldn't point to `__init__.py`. I'm getting this from the [paga docs](https://scanpy.readthedocs.io/en/latest/api/scanpy.tl.paga.html).

It would be terrific if you could make this work again: I suspect it's a small change for you...

**Comment by falexwolf** [comment]
We already had this issue 2.5 years ago: https://github.com/theislab/scanpy/issues/36, at the time, of course, with a very different docs backend.

**Comment by flying-sheep** [comment]
Well, `project_dir` is configurable:

https://github.com/theislab/scanpydoc/blob/02a0fcb5b5ddfd1f9427c27e736e83126f6cfc64/scanpydoc/rtd_github_links.py#L144

but why does it go to `__init__.py`?

**Comment by flying-sheep** [comment]
I think the issue we had 1½ years ago was that we didn’t handle the links at all!

**Comment by flying-sheep** [comment]
Fixed in theislab/scanpydoc@10fec8a371398a2e0ecdd0a912fd10b13b11fe1f

**Comment by falexwolf** [comment]
Great! :smile:
